astropy/coordinates/frame_attributes.py

```diff
@@ -315,7 +315,7 @@ def convert_input(self, value):
             return value, converted
 
 
-class EarthLocationAttribute(FrameAttribute):
+class EarthLocationFrameAttribute(FrameAttribute):
```

Comment:
So that we don't break backwards compatibility, please add subclasses to the two new classes that raise a deprecation warning in the initializer and pass through any arguments. You can use the [`AstropyDeprecationWarning`](http://docs.astropy.org/en/latest/api/astropy.utils.exceptions.AstropyDeprecationWarning.html?highlight=astropy%20deprecation) class.